### PR TITLE
provider/openstack: structuring of the documentation sidebar

### PR DESCRIPTION
**Pull request by berendt**

### CODE REVIEW

File: website/source/layouts/openstack.erb [hunk 1]
```diff
@@ -64,11 +58,48 @@
             <li<%= sidebar_current("docs-openstack-resource-networking-subnet-v2") %>>
               <a href="/docs/providers/openstack/r/networking_subnet_v2.html">openstack_networking_subnet_v2</a>
             </li>
+          </ul>
+        </li>
+
+        <li<%= sidebar_current(/^docs-openstack-resource-lb/) %>>
+          <a href="#">Loda Balancer Resources</a>
```
Comment (apparentlymart) [inline website/source/layouts/openstack.erb]: Did you mean "Load Balancer" here?

Comment (berendt) [inline website/source/layouts/openstack.erb]: Of course. I will fix this.

**Comment by apparentlymart** [comment]
:+1:

**Comment by jtopjian** [comment]
:+1:

**Comment by catsby** [comment]
Excellent, thank you!

**Pull request merged**